code/_compile_options.dm

```diff
@@ -1,4 +1,4 @@
-//#define TESTING //By using the testing("message") proc you can create debug-feedback for people with this
+#define TESTING //By using the testing("message") proc you can create debug-feedback for people with this
```

Comment:
Don't forget to comment out the testing stuff.

Reply:
Noted